README.md

```diff
@@ -1,39 +1,62 @@
 Instagram Challenge
 ===================
+Build Instagram using Rails.
 
-## Instructions
+## User stories
 
-* Feel free to use Google, your notes, books, etc., but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
+As a user,  
+So my content is identifiable,  
+I want to be able to **sign-up**.
 
-## Task
+As a user,    
+So I can show off a photo/image,  
+I want to be able to **post pictures** from my account.
 
-Build Instagram: Simple huh!
+As a user,
+So I can be confident other users won't post as me,
+I want to be able to **login and logout**.
 
-Your challenge is to build Instagram using Rails. You'll need **users** who can post **pictures**, write **comments** on pictures and **like** a picture. Style it like Instagram's website (or more awesome).
+As a user,  
+So I can engage with other users' content,  
+I want to be able to **post comments** about pictures.
 
-Bonus if you can add filters!
+As a time-pressed user,  
+So I can show love for other users' content,  
+I want to be able to **like** a picture with one click.
 
-## How to start
+Nice-to-haves:  
+- Style it like Instagram's website (or more awesome).
+- Add filters
 
-1. Produce some stories, break them down into tasks, and estimate
-2. Fork this repo, clone, etc
-3. Initialize a new rails project
+## Getting started
+To get started with the app, clone the repo and then install the needed gems:
+```
+$ bundle install --without production 
+```
+Next, migrate the databases:
+```
+$ rails db:migrate
+```
+Finally, run the test suite to verify that everything is working correctly:
+```
+$ rails test
```

Comment:
Clear instructions - one thing I would mention is to run 'yarn' or 'rails webpacker:install' as when I tried to run tests I had a couple of errors saying 'Webppacker cannot find application.js..', but running the above command solved the issue for me!